packages/@react-spectrum/calendar/test/Calendar.test.js

```diff
@@ -418,6 +420,8 @@ describe('Calendar', () => {
       expect(selectedDate).toHaveFocus();
 
       fireEvent.keyDown(grid, {key: 'ArrowRight'});
+      fireEvent.keyUp(document.activeElement, {key: 'ArrowRight'});
```

Comment:
because focus can move on the key down, in which case the original element would not be the recipient of the key up. this just ensures that
I could also use document.activeElement for both, but I wasn't in the mood to change all of them